src/rgw/rgw_coroutine.cc

```diff
@@ -627,6 +627,7 @@ int RGWCoroutinesManager::run(RGWCoroutine *op)
     r = op->get_ret_status();
   }
   op->put();
+  delete stack;
```

Comment:
very skeptical about this one. `RGWCoroutinesStack` is a ref-counted structure, and we _should_ be calling `put()` on each stack as it completes in the `RGWCoroutinesManager::run(list<RGWCoroutinesStack *>& stacks)` overload above